### ISSUE TITLE
Removal of RegionOfInterest from calculations

### ISSUE THREAD

**Issue by pomodoren** [dbbs-lab/cerebellar-models#2]
Based on the JSONread previously we have this:
```
class BrainRegionTree:
    def __init__(self, region, ...):
         self.region_of_interest = 'Flocculus'  # roi
         self.involved_regions = ['Purkinje', 'granular', 'molecular']
```
So, we remove the region of `Flocculus` from our list of included regions. 

I (quickly) tested whether it affects our results, and it `Doesnt`, because `Flocculus` does not have any representation in `ann` (annotations of the voxels to regions). (check `tests/unit/test_brain_atlas_input.test_leaf_or_next`)

Still other bigger regions do have representations that are represented by their ids only, and not by their children's ids. 
I am guessing that when doing `INVOLVED REGIONS` in analysis, we will later include the region itself too, because their children do not represent the whole region.

In other words:
`sum(ann==big_area.id) > sum([sum(ann==child.id) for  child in big_area.children])`
